cabal-install/src/Distribution/Client/ProjectBuilding/Types.hs

```diff
@@ -173,9 +175,10 @@ type BuildOutcome  = Either BuildFailure BuildResult
 -- | Information arising from successfully building a single package.
 --
 data BuildResult = BuildResult {
-       buildResultDocs    :: DocsResult,
-       buildResultTests   :: TestsResult,
-       buildResultLogFile :: Maybe FilePath
+       buildResultDocs      :: DocsResult,
+       buildResultTests     :: TestsResult,
+       buildResultLogFile   :: Maybe FilePath,
+       buildResultBuildInfo :: Maybe Text
```

Comment:
Unused, also what do these BuildResults really do?

Reply:
Good question. I'd like this answered too

Reply:
It seems like they were *intended* to be cached somehow, but it kind of looks like they are always created on the fly.
Probably going to re-activate the usage in favour of the cache files in `dist-newstyle/cache/buildinfo`.